.github/workflows/ci.yml

```diff
@@ -18,7 +18,12 @@ jobs:
     strategy:
       matrix:
         build_type: [Debug, Release]
-        os: [ubuntu-latest]
+        os: [ubuntu-latest, macOS-10.14]
+        include:
+          - os: ubuntu-latest
+            cmake_generator: "Unix Makefiles"
+          - os: macOS-10.14
+            cmake_generator: "Xcode"
 
```

Comment:
This is something of GHA that I haven't yet understood. What is the effect of the `include` entry? There is already a 2x2=4 matrix from the `build_type` and `os` tags, would the include add two more (as in Travis CI if I recall?) resulting into 6 jobs? It seems that only 4 start.

Reply:
As mentioned in the GitHub docs (in https://help.github.com/en/articles/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix): 
> You can add additional configuration options to a build matrix job that already exists. You cannot use include to add new jobs to a build matrix. For example, if you want to use a specific version of npm when the job that uses windows-2016 and version 4 of node runs, you can use include to specify that additional option.

Basically the include option does not add jobs, but just additional options to existing jobs.

Reply:
Ow this is cool, I was not aware of it. Then, job exclusion can be done with job-wise `if`s.